message TAdCatProto {
    optional bool Explicit   = 1;
    optional bool Perversion = 2;
    optional bool Sensitive  = 3;
    optional bool Grey       = 4;
    optional bool Gruesome   = 5;
    optional bool Child      = 6;
    optional bool Ordinary   = 7;
    optional bool Fixlist    = 8;
}

enum EAdCat {
    AC_EXPLICIT   = 0;
    AC_PERVERSION = 1;
    AC_SENSITIVE  = 2;
    AC_GREY       = 3;
    AC_GRUESOME   = 4;
    AC_CHILD      = 5;
    AC_ORDINARY   = 6;
    AC_FIX_LIST   = 7;
}
